yarn.lock

```diff
@@ -367,6 +367,15 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@babel/runtime@npm:^7.24.5":
```

Comment:
I think `yarn dedupe` might clean up some of these additions.

Reply:
Thanks for suggestion! deduped in https://github.com/grafana/plugin-ui/pull/137/commits/5de2ced29635ee376fbd02c72ce11d447da66da6